src/pages/index.js

```diff
@@ -175,12 +205,87 @@ const editProfilePopup = new PopupWithForm(
 
 editProfilePopup.setEventListeners();
 
-/// UserInfo class ///
+const addPlacePopup = new PopupWithForm(
+  { popupSelector: "#profile__add-form" },
+  handleAddFormSubmit
+);
 
-const newUserInfo = new UserInfo({
-  // llamo a la clase para usar los metodos
-  nameSelector: "#profile__title", // los selectores que use, nose porque no llevan -input al final.
-  jobSelector: "#profile__description",
+addPlacePopup.setEventListeners();
+
+//// PopupWithImage class ////
+
+const popupImage = new PopupWithImage("#preview__image_modal"); //llamo a la clase (insatntiate)
+
+///submit functions  (add, edit, profilepicture )  ///
+
+function handleAddFormSubmit(inputValues) {
+  const { name, link } = inputValues; // Extract name and link from input values
+  addPlacePopup.setButtonText("Saving.."); // cree la funcion con el popUpWithForm y pase text como parametro
+  apiInstance
+    .addNewCard(name, link) // toma el name y el link
+    .then((newCard) => {
+      // newCard es la respuesta y contiene el name y link
+      console.log(newCard);
+
+      const cardElement = getCardView(newCard); // usamos esa respuesta y la ponemos en una constante junto con la funcion que genera la carta
+      cardSection.addItem(cardElement); // mandamos la carta recien creada con la info de API al browser o UI
+
+      addPlacePopup.close(); // cerramos el popup
```

Comment:
The api request looks correctly, everything are on the right places

Reply:
thank you !!